docs/getting-started/v3-migration.md

```diff
@@ -86,6 +86,7 @@ Chart.js 3.0 introduces a number of breaking changes. Chart.js 2.0 was released
 * `_model.label`
 * `Point._model.tension`
 * `Point._model.steppedLine`
+* `TimeScale._getPixelForOffset`
 * `TimeScale.getLabelWidth`
```

Comment:
Can we add that the `TimeScale.getPixelForValue` now expects the parameter to be in milliseconds

Reply:
done